app/models.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+
+from flask_sqlalchemy import SQLAlchemy
+
+db: SQLAlchemy = SQLAlchemy()
+
+
+class Account(db.Model):
+    """
+    Model for table account
+    """
+
+    id = db.Column(db.Integer(), primary_key=True, autoincrement=True)
+    username = db.Column(db.String(length=100), unique=True)
+    password = db.Column(db.String(length=500))
+    email = db.Column(db.String(length=100))
```

Comment:
these must not be nullable (they are by default)

Reply:
done